WISE-Community/WISE-Client
# feat(Unit Library): Allow archiving units

@breity Can you take a look at the styling and make changes if necessary?

## Changes
- Teachers can archive units in the My Units section
- Added Archive/Restore option in the unit info popup 3 dots menu
- Added checkbox to each unit card
- Added global checkbox to select all units in view
- Added global Archive/Restore button to archive/restore multiple units at once
- Added drop down to switch between Active and Archived view

## Test
- Test with https://github.com/WISE-Community/WISE-API/pull/248
- Make sure you can archive/restore a single unit using the unit info popup 3 dots menu
- Make sure you can archive/restore multiple units at once
- Make sure you can switch between Active and Archived view
- When you select multiple units, make sure the count of the selected units is displayed
- If units are selected and you switch between Active/Archived view or go to the next page of units, all the units become unselected

Closes #1549
